src/lib.rs

```diff
@@ -44,7 +44,7 @@ mod session;
 use std::time::Duration;
 
 pub use error::{Error, Result};
-pub use row::{SnowflakeDecode, SnowflakeRow};
+pub use row::{SnowflakeColumn, SnowflakeColumnType, SnowflakeDecode, SnowflakeRow};
```

Comment:
Since we don't guarantee the behavior of these structs, let's not export them for now. If we want to export them in the future, we can do so without breaking backward compatibility.